keepassxc-browser/background/page.js

```diff
@@ -28,7 +28,9 @@ const defaultSettings = {
     showOTPIcon: true,
     useObserver: true,
     usePredefinedSites: true,
-    usePasswordGeneratorIcons: false
+    usePasswordGeneratorIcons: false,
+    webAuthn: false,
+    webAuthnFallback: false,
```

Comment:
The options text says it is enabled by default. Although, when would you _not_ want to fallback? Does it make sense to even offer this option?

Reply:
The idea behind this option was a situation where KeePassXC is open and the authentication/register goes there, but the user wants to cancel it and use the "normal" hardware method instead. When the fallback is disabled, the whole operation is just canceled and it expects KeePassXC to be available. Just wanted to give an extra option for this kind of scenario.